ExpenseTracker/ExpenseTracker.Application/Stores/WalletStore.cs

```diff
@@ -144,8 +144,9 @@ await _hubContext.Clients.All
 
             if (user is null)
             {
-                var message = new EmailMessage(userEmail, "there", "Collaboration Invitation", null);
-                _emailService.SendWalletInvitation(message);
+                var inviteSenderName = wallet.Owner.Email;
+                var message = new EmailMessage(userEmail, "", "Collaboration Invitation", null);
```

Comment:
I think _username_ should be given to the second parameter of the **EmailMessage** method

Reply:
> > I think _username_ should be given to the second parameter of the **EmailMessage** method
> 
> hali ro'yhatdan o'tmagan kishining qanday qilib UserName ni olamiz

Oops! my mistake, I think that the registered user is being invited